pages/textbook-beta/course/quantum-states-and-qubits.vue

```diff
@@ -33,12 +33,24 @@ export default class QuantumStatesAndQubitsPage extends CourseOverviewPage {
     }
   }
 
-  references: string[] = [
-    'Michael Nielsen and Isaac Chuang. Quantum Computation and Quantum Information (Chapter 1). Cambridge University Press, 2000.',
-    'Phillip Kaye, Raymond Laflamme, and Michele Mosca. An Introduction to Quantum Computing (Chapters 1-3). Oxford University Press, 2007.'
-  ]
-
+  references: string[] = []
   links: GeneralLink[] = [
+    {
+      url: 'http://mmrc.amss.cas.cn/tlb/201702/W020170224608149940643.pdf',
+      label: 'Michael Nielsen, et all. Quantum Computation and Quantum Information (Chapter 1). Cambridge University Press, 2000.',
+      segment: {
+        cta: 'mlmastery-linear-algebra',
+        location: 'external-recommended-readings'
+      }
+    },
+    {
+      url: 'http://mmrc.amss.cas.cn/tlb/201702/W020170224608149125645.pdf',
+      label: 'Phillip Kaye, et all. An Introduction to Quantum Computing (Chapters 1-3). Oxford University Press, 2007.',
+      segment: {
+        cta: 'mlmastery-linear-algebra',
```

Comment:
for this one, can we update this CTA value to something like `introduction-to-quantum-computing-ch-1-3`